datadog_checks_dev/setup.py

```diff
@@ -58,11 +58,14 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Do we "officially" support 3.8 for `ddev` already? I imagine tests ran fine locally which is why you added this classifier…

Reply:
Yup!